resources/5/ajax.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="hu">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Web-programozás-1 Előadás Házi feladat - AJAX</title>
+    <link type="text/css" rel="stylesheet" href="../main.css">
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div id="pageContainer">
+        <div id="contentWrap">
+            <header>
+                <h1 class="header">Web-programozás-1 Előadás Házi feladat</h1>
+            </header>
+            <nav>
+                <ul>
+                    <li><a href="../../index.html">Főoldal</a></li>
+                    <li><a href="../2/tablazat.html">Táblázat</a></li>
+                    <li><a href="../3/html5.html">HTML5</a></li>
+                    <li><a href="../4/chartjs.html">ChartJS</a></li>
+                    <li><a href="ajax.html" class="active">AJAX</a></li>
+                    <li><a href="../oojs.html">OOJS</a></li>
+                </ul>
+            </nav>
+            <aside>
+                <h1>AJAX alkalmazás</h1>
+                <hr>
+                <h2>Az 5. feladat megvalósítása</h2>
+            </aside>
+            <section>
+                <article>
+                <h1>AJAX alkalmazás</h1>
+                </article>
+                <div class="container">
+                    <div class="box">
+                        <p id="code">code= VE22CSabc123</p>
+                        <h3>Adatok lekérése</h3>
+                        <button id="readBtn">Adatok lekérése</button>
+                        <div id="readDiv"></div>
+                    </div>
+                    <div class="box">
+                        <h3>Magasság Statisztika</h3>
+                        <p>Összeg: <span id="sumHeight">-</span></p>
+                        <p>Átlag: <span id="avgHeight">-</span></p>
+                        <p>Legnagyobb: <span id="maxHeight">-</span></p>
+                    </div>
+                    <div class="box">
+                        <h3>Új adat létrehozása</h3>
+                        Név (kötelező, max 30):<br><input id="name1" type="text"></input><br>
```

Comment:
Can you please rename "name1" to "newNameInput", update the id in the javascript as well!